### PR TITLE
update fm-info.json

### PR DESCRIPTION
**Pull request by bwhm**
@DzhideX is not here to review, so please double check it loads properly :)

### CODE REVIEW

**Review by blrhc — APPROVED** [review]

File: data/fm-info.json [hunk 1]
```diff
@@ -10,823 +10,847 @@
...
+              {"memberHandle":"0x2bc","memberId":2098,"inducted":null,"directScore":500,"referrer":null},
+              {"memberHandle":"botzmann","memberId":604,"inducted":null,"directScore":475,"referrer":null},
+              {"memberHandle":"olgakuzia","memberId":2345,"inducted":null,"directScore":475,"referrer":{"memberHandle":"razumv","memberId":1019,"inducted":null,"referralScore":[0,0,0,0,0,0,0,0,0,0,95]}}
           ]
         }
     },
     "scores":
     {
-        "cutoff": "2021-06-24T14:09:34.379Z",
+        "cutoff": "2021-07-13T14:42:56.253Z",
```
Comment (blrhc) [inline data/fm-info.json:35]: @bwhm Did notice something, I stopped after I got to 120 hours after the period ended, so the cutoff is actually 2021-07-11 at 00:00.
```suggestion
        "cutoff": "2021-07-11T00:00:00.000Z",
```

File: data/fm-info.json [hunk 2]
```diff
@@ -902,72 +926,108 @@
...
+            "extraAllocation": 0,
+            "directScores": [0,175,490,650,400,650,750,750,750,1675,1850],
+            "totalDirectScore": 8140,
+            "referralScores": [],
+            "totalReferralScore": 0,
+            "referrer": {"memberHandle":"leet_joy","memberId":957,"inducted":null,"referralScore":[0,35,98,130,80,130,150,150,150,335,370]},
+            "totalScore": 8140
+        },
+        {
+            "memberHandle": "@maxlevush",
```
Comment (blrhc) [inline data/fm-info.json:1021]:
```suggestion
            "memberHandle": "maxlevush",
```

File: data/fm-info.json (hunk 1 above)
Comment (blrhc) [reply data/fm-info.json:35]: Not sure how to do midnight in this format, correct if wrong @bwhm!

**Review by blrhc — CHANGES_REQUESTED** [review]
see comments above

**Comment by bwhm** [comment]
Updated with the exact answer :)

**Pull request merged**